api/src/main/resources/client_api.yaml

```diff
@@ -179,7 +186,33 @@ definitions:
       orderBy:
         description: >
           An array of columns to sort the resulting data by, taken from the table specified above,
-          each one optionally followed by " DESC" for descending sort order. (Default varies by table.)
+          each one optionally enclosed in "DESCENDING()" for descending sort order.
+          (Default sort order is person_id followed by the ID of the specified table.)
+        type: array
+        items:
+          type: string
+
+  AnnotationQuery:
+    type: object
+    description: >
+      A specification for retrieving annotation values or review statuses entered for participants.
+    properties:
+      annotations:
+        description: >
+          An array of names of annotations to retrieve about participants, "review_status" for
+          the cohort review status of the participant, or "person_id" for the
+          ID of the participant. Defaults to "person_id", "review_status", and the names of all
+          defined annotations in the cohort review. This is only valid in combination with a named
+          cohort that has a cohort review created for it; if no cohort review has been created,
+          no results will be returned.
```

Comment:
I'm assuming this means that cohort_name has to be set on the MaterializeCohortRequest? and that the query will return data on all participants in the review or the entire cohort definition? Will you be able to query by person_id or only cohort_name?

Reply:
Correct, you have to use cohortName; and it will only return data on the participants in the review. I updated the documentation to say this.

You can only query by cohortName in this API. (You could of course filter the results in memory down to a particular participant... but normally I don't think you'd want to do this sort of thing in a notebook.)

I think if we decide we want to use the code I'm writing in the cohort review UI (if that's what you were thinking of?), we're probably going to want to expose a different endpoint that returns data for a single participant (with the ID passed up in the request)... we can refactor the existing code to make that pretty simple to implement.